drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -472,6 +473,21 @@
   font-size: 0.875em;
 }
 
+.xblock--drag-and-drop .btn-link {
+    padding: 1px;
+    font-size: 0.875em;
+}
+
+.xblock--drag-and-drop .btn-default.btn-link:hover,
```

Comment:
@cahrens Beccause this css is needed to override `btn-default` styling. Without this CSS the button that supposed to look like a link gets blue border on hover. If we remove `btn-default` rules controlling hover styling (they're exactly below), than all the other buttons loose blue border on hover, which they suppose to have.

Completely removing custom button styling and relying on edX styling is not a good idea too, as (1) it would require significant overhaul of styling and (2) DnDv2 is used with non-edX LMS, so removing custom styles will introduce regressions there.